lib/engine/game/g_18_rhl/step/lay_tile_checks.rb

```diff
@@ -5,5 +5,13 @@ def lay_tile(action, extra_cost: 0, entity: nil, spender: nil)
     super
 
     @game.eastern_ruhr_connection_check(action.tile.hex)
+    return if action.tile.hex.name != 'E14' || !@game.prinz_wilhelm_bahn
+
+    # Remove hex block
+    comp = @game.prinz_wilhelm_bahn
+    comp.all_abilities.each do |a|
```

Comment:
do you need to dup here or no

Reply:
Probably safe in this case as it only contains one ability... but sure adding dup is fine, so I am doing that.